src/components/text-editor/text-editor.tsx

```
@@ -224,44 +224,39 @@ export class TextEditor implements FormComponent<string> {
-                }}
+        return [
+            <limel-notched-outline
+                labelId={this.editorId}
+                label={this.label}
+                required={this.required}
+                invalid={this.invalid}
+                disabled={this.disabled}
+                readonly={this.readonly}
+                hasValue={!!this.value}
+                hasFloatingLabel={true}
             >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)